Shops/ShopGoblinTinkerer.cs

```diff
@@ -134,234 +118,216 @@ public void InitShop(string currentShop)
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.SandstorminaBottle);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.TsunamiInABottle);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                     }
                     if (NPC.downedBoss3)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.Aglet);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.ClimbingClaws);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.AnkletoftheWind);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.Flipper);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.ShoeSpikes);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.FlyingCarpet);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.DivingHelmet);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.ShinyRedBalloon);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.LuckyHorseshoe);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                         shop.item[nextSlot].SetDefaults(ItemID.LavaCharm);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
+                        
                     }
                     if (NPC.downedPlantBoss)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.Tabi);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
```

Comment:
You need `++` otherwise everything will be merged into one slot.

Please review the prices.